docs/events.mdx

```diff
@@ -1,10 +1,8 @@
-Asynchronous messages (events) are one of the primary ways Microservices and Functions communicate. Nitric provides an interface to message-oriented middleware and message brokers to facilitate low-latency, highly available, "atleast once" message delivery.
+Nitric's events service and API provides an intuitive approach to asynchronous messaging allowing services, functions and more to communicate via topics with event messages. While synchronous request/response communication is quite common, events provide a more reliable mechanism for communication in distributed and serverless systems, with the added benefit that services can be decoupled from each other through a [publish/subscribe](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern) relationship.
 
-<img
-  src="../assets/img/topics-overview.svg"
-  height="334"
-  alt="topic overview diagram"
-/>
+The Nitric events API provides an interface to message-oriented middleware and message brokers such as AWS SNS & Google PubSub to facilitate low-latency, highly available, "at-least once" message delivery.
+
+<img src="/img/topics-overview.svg" height="334" alt="topic overview diagram" />
```

Comment:
The image src needs to point to the assets folder. This is normalized by the website.